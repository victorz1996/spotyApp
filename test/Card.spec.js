import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Bootstrapvue from 'bootstrap-vue'
import card from '../components/Card.vue'

let wrapper
beforeEach(() => {
  Vue.use(Bootstrapvue)
  wrapper = shallowMount(card, {
    propsData: {
      item: {
        images: [],
      },
    },
  })
})

describe('Card', () => {
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it('HTMl renderizado structure esperada', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
