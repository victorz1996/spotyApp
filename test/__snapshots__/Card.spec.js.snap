// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card HTMl renderizado structure esperada 1`] = `
<div>
  <b-card-stub
    bodytag="div"
    class="mb-2 puntero"
    footertag="div"
    headertag="div"
    imgsrc="no-image.png"
    imgtop="true"
    style="max-width: 20rem;"
    subtitletag="h6"
    subtitletextvariant="muted"
    tag="div"
    titletag="h4"
  >
    <b-card-text-stub
      texttag="p"
    />
  </b-card-stub>
</div>
`;
